packages/fxa-content-server/app/scripts/views/sign_up_password.js

```diff
@@ -130,10 +130,6 @@ const SignUpPasswordView = FormView.extend({
         });
       }
 
-      // returns null if coppa enabled, otherwise will use submitted age
-      // to determine if age is >=18 at time of sign up
-      account.set({ atLeast18AtReg: this.isAgeAtLeast18() });
```

Comment:
I think we'll need to either set it on the model but not persist, or pass the atLeast18AtReg value directly to the signUp function (else the value won't be saved to the db)

Reply:
Update: tested locally with _only_ the changes to `account.js`, and:
- atLeast18AtReg not saved to localStorage :white_check_mark: 
- atLeast18AtReg in db :white_check_mark: